livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java

```diff
@@ -131,6 +132,12 @@ public Integer createSession(InterpreterContext context, String kind) throws Exc
     }
   }
 
+  protected void initializeSpark(final InterpreterContext context,
+                                 final Map<String, Integer> userSessionMap) throws Exception {
+    interpret("val sqlContext = new org.apache.spark.sql.SQLContext(sc)\n" +
+        "import sqlContext.implicits._", context, userSessionMap);
```

Comment:
yes, as LivySQLInterpreter needs SQLContext. For other interpreters (livy.spark, livy.pyspark, livy.sparkr), if SQLContext is not available, user can create it by himself.